src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,79 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { getFilteredPeople } from '../utils/GetFilteredPeople';
+import { getPeopleWithParents } from '../utils/GetPreparedPeople';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query');
+  const centuries = searchParams.getAll('century');
+  const sex = searchParams.get('sex');
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const visiblePeople = getFilteredPeople(
+    people,
+    {
+      centuries: centuries,
+      sex: sex,
+      query: query,
+    },
+    { sort, order },
+  );
+
+  useEffect(() => {
+    setLoading(true);
+    setLoadingError(false);
+
+    getPeople()
+      .then(humans => {
+        setPeople(getPeopleWithParents(humans));
+      })
+      .catch(() => setLoadingError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+          {!loading && people.length && (
```

Comment:
The condition `!loading && people.length` should be `!loading && !!people.length` to ensure it evaluates to a boolean. This prevents potential issues if `people.length` is zero, which would cause the condition to evaluate to false.